app/utils/DateUtils.scala

```diff
@@ -6,6 +6,9 @@ import org.joda.time.{DateTime, Days}
 import play.api.libs.json.{Format, Reads, Writes}
 
 object DateUtils {
+
+  val freshnessLimit = 30
```

Comment:
It is exposed and can be used elsewhere. In this case, it is used to show the green goal threshold in the percentiles graph. It is not absolutely necessary though